tests/post_training/test_templates/test_smooth_quant.py

```diff
@@ -168,7 +168,7 @@ def test_get_abs_max_channel_collector(self):
         ),
     )
     # pylint:disable=protected-access
-    def test__get_nodes_to_smooth_data(self, model_cls, references, tmpdir):
+    def test_get_nodes_to_smooth_data(self, model_cls, references, tmpdir):
```

Comment:
The method has the `_get_nodes_to_smooth_data` name. This is why the test has double underlines.